rust/worker/src/sysdb/sysdb.rs

```diff
@@ -0,0 +1,237 @@
+use async_trait::async_trait;
+use uuid::Uuid;
+
+use crate::chroma_proto;
+use crate::types::{CollectionConversionError, SegmentConversionError};
+use crate::{
+    chroma_proto::sys_db_client,
+    errors::{ChromaError, ErrorCodes},
+    types::{Collection, Segment, SegmentScope},
+};
+use thiserror::Error;
+
+const DEFAULT_DATBASE: &str = "default_database";
+const DEFAULT_TENANT: &str = "default_tenant";
+
+#[async_trait]
+pub(crate) trait SysDb: Send + Sync + SysDbClone {
+    async fn get_collections(
+        &mut self,
+        collection_id: Option<Uuid>,
```

Comment:
To confirm I understand correctly: We accept this parameter because it's in the proto, right? And in practice we use `GetCollections` to also just get a single collection

Reply:
Yeah